src/tests/EdFi.DataManagementService.Tests.E2E/Features/SchoolYearReferenceValidation.feature

```diff
@@ -0,0 +1,136 @@
+Feature: School Year Reference Validation
+  Reference validation on School Years
+
+        Background:
+            Given the Data Management Service must receive a token issued by "http://localhost"
+              And user is already authorized
+             When a POST request is made to "/ed-fi/schoolYearTypes" with
+                  """
+                    {
+                      "schoolYear": 2024,
+                      "currentSchoolYear": true,
+                      "schoolYearDescription": "Current School Year 2024"
+                    }
+                  """
+             Then it should respond with 201 or 200
+              When a POST request is made to "/ed-fi/schoolYearTypes" with
+                  """
+                    {
+                      "schoolYear": 2029,
+                      "currentSchoolYear": true,
```

Comment:
Not important to the test, but you should only set one of these currentSchoolYears to true